misk-events/src/main/kotlin/misk/events/SpooledProducer.kt

```diff
@@ -12,6 +12,20 @@ import misk.hibernate.Session
  * to the event stream, and are done so through a [Producer] transaction.
  */
 interface SpooledProducer {
+  /**
+   * Publishes an event to a sharded spool.
+   *
+   * @param session Session for the database transaction the event should be spooled in.
+   * @param groupRootId Entity group root ID for the event, used as the sharding key in the database,
+   *   and will be passed to the mapping function configured for the spool to look up a partition
+   *   key for the event.
+   */
   fun publish(session: Session, groupRootId: Gid<*, *>, topic: Topic, vararg event: Event)
+
+  /**
+   * Publishes an event to a sharded spool.
+   *
+   * @param session Session for the database transaction the event should be spooled in.
+   */
   fun publishUnsharded(session: Session, topic: Topic, vararg event: Event)
```

Comment:
worthwhile documenting the rest of the params?

Reply:
No, I don't think so.